lobby.go

```diff
@@ -34,6 +34,7 @@ var (
 		MinClientsPerIPLimit: 1,
 		MaxClientsPerIPLimit: 24,
 	}
+	supportedLanguages = []string{"English", "Italian"}
```

Comment:
These should be constants, as they are also used at:

https://github.com/scribble-rs/scribble.rs/blob/8d4c820afe821dff9f37579644346191036a9376/words.go#L16